packages/sdk-cli/src/models/localRelay/util.test.ts

```diff
@@ -0,0 +1,42 @@
+import { promises as fsPromises } from 'fs';
+import { isInt, readJsonFile } from './util';
+
+describe('isInt', () => {
+  describe('false', () => {
+    it.each([undefined, null, Infinity, -Infinity])('%s', (x) =>
+      expect(isInt(x)).toBe(false),
+    );
+
+    it('number string', () => expect(isInt('5')).toBe(false));
+  });
+
+  describe('true', () => {
```

Comment:
NIT: Maybe to be more specific add a few more cases here. Examples that spring to mind are:
* Float
* Negative number

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/ce942bc655511107cf04d27c00c93403e9ae3c83